src/discussions/posts/data/slices.js

```diff
@@ -57,7 +57,6 @@ const threadsSlice = createSlice({
       if (state.author !== payload.author) {
         state.pages = [];
         state.author = payload.author;
-        state.totalThreads = null;
```

Comment:
I feel like this fix will have ripple issues as  this is emptying older threads count when a new request is made and if at certain point the call will fail, redux will still be populated with outdated data